tools/run_tests/python_utils/port_server.py

```diff
@@ -31,7 +31,7 @@
 # increment this number whenever making a change to ensure that
 # the changes are picked up by running CI servers
 # note that all changes must be backwards compatible
-_MY_VERSION = 20
+_MY_VERSION = 21
```

Comment:
Optional: Is there any sort of sanity test for this? I absolutely would have missed this update if I were doing this PR myself. I'm thinking something like [`IfChange`](https://engdoc.corp.google.com/eng/doc/tott/episodes/439.md?cl=head)

Reply:
There's no sanity test for this and also it's mostly a legacy thing - our kokoro jobs get a new VM each time a build runs, so they always end up with the freshest version of port server anyway (This used to be a big deal where our test VMs weren't ephemeral so we needed to make sure they have an up-to-date version of port server).

Right now the only thing it does is restart port server on people's workstation (if they are still using run_tests.py - which we are slowly trying to get rid of), so I'll leave as is.